Assets/Tests/InputSystem/CoreTests_State.cs

```diff
@@ -758,6 +758,17 @@ public void State_StateChangeMonitorsStayIntactWhenOtherDevicesAreRemoved()
         Assert.That(positionMonitorFired);
     }
 
+    [Test]
+    [Category("State")]
+    public void State_CurrentTimeCalculatedCorrectly()
```

Comment:
Plz avoid things like `xxxCorrectly` and `xxxWorks` kind of tests. From these, it's never quite evident what "correct" implies.

For this test, for example, something like `State_CurrentTimeTakesOffsetToRealtimeSinceStartupIntoAccount` would advertise more clearly what the behavior is that the test wants to establish.

Reply:
Fixed the test name, thanks